BlogTemplate/wwwroot/js/slugUpdateWarning.js

```diff
@@ -0,0 +1,18 @@
+(function () {
+
+    var editpostform = document.querySelector("#editpost");
+
+    editpostform.addEventListener("submit", function (e) {
+        var titleElm = this.querySelector("#newPost_Title");
+        var oldtitle = titleElm.getAttribute("data-oldtitle");
+        var newtitle = titleElm.value;
+
+        if (oldtitle !== newtitle) {
+            if (confirm("Changing the post title will update the post slug and break external links. \r\rDo you wish to update the slug?")) {

```

Comment:
When I click "Cancel" in the pop-up, I get a null-reference exception. The error is coming from line 7 in Post.cshtml, but I think something in this file or a related file is causing the problem.